.travis.yml

```diff
@@ -6,12 +6,23 @@ php:
   - 5.6
   - hhvm
 
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: GUZZLE_VERSION=4.*
+    - php: 5.6
+      env: GUZZLE_VERSION=5.*
+    - php: 5.6
+      env: GUZZLE_VERSION=6.*
```

Comment:
the normal PHP 5.6 build already runs on Guzzle 6, and the 5.4 one runs on guzzle 5 (as 6 is not compatible). So you only need to add a job for Guzzle 4, making the matrix smaller